frontend/src/Pages/Student/Result/CourseResults.jsx

```
@@ -0,0 +1,213 @@
+                    <div className="mb-4 text-center">
+                        <p className="text-lg font-semibold">
+                            Semester CGPA: <span className="text-blue-600">{cgpa?.toFixed(2)}</span>
+                        </p>
+                    </div>
+
+                    {/* PDF Download Button */}
+                    <div className="flex justify-center">
+                        <button
+                            onClick={downloadPDF}
+                            className="px-6 py-2 font-bold text-white bg-green-500 rounded-md hover:bg-green-600 focus:outline-none focus:ring-2 focus:ring-green-500"
+                        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)